### DISCUSSION TITLE
Adobe Live Search and swissup/breeze-blank 2.19.0

### DISCUSSION THREAD

**Discussion by phpDevChandra** [breezefront/community#342] [5. General]
HI,

I am upgrading Magento commerce from 2.4.6-p7 to 2.4.7-p3 with composer version 2.7 and PHP 8.3 with  Live search.

We are getting issue during search with latest swissup/breeze-blank 2.19.0 and  swissup/theme-frontend-breeze-blank 2.9.0 while its working well the previous version 2.18.0. See attached.
![search issue with breeze 2 19 0](https://github.com/user-attachments/assets/6a2f49a9-c0c4-443a-80e2-03569e6208d1)

We can see error in console there is 400 Bad request.Not sure if that is cause. See atatched -
![when we click on bad request](https://github.com/user-attachments/assets/80b0ddaa-5e0f-4d4d-9d04-c60954460224)

In log fiels we can see the error -
![Error in log](https://github.com/user-attachments/assets/86e38300-af79-45ff-a19d-fde87e06e60b)

Its working well in LUMA and when I repalced swissup/breeze-blank 2.19.0 to 2.18.0 and swissup/theme-frontend-breeze-blank 2.8.5 then works well.

Can you please help on this.

Thanks,

**Comment by phpDevChandra** [comment]
Hi,

I dig it at my side and found the cause.

Issue is with **swissup/theme-frontend-breeze-blank 2.9.0** in Magento_Theme/layout/default.xml

A piece of code added seems for accessibilty not sure. See the below screenshot-
![Screenshot from 2024-12-09 14-52-58](https://github.com/user-attachments/assets/1ec62912-7599-426f-b178-d315e59bd0b4)

I will manage it by  a patch but please look into this.

Thanks..

**Reply by vovayatsyuk** [reply]
Hello,

Yes, this code was added for keyboard accessibility (DOM order should match visual representation). Unfortunately, I don't know why it's causing an issue with the Adobe Live Search module and I don't have AdobeCommerce to test.

You can move the sidebar back using the following layout XML instruction in your theme:

```xml
<move element="div.sidebar.main" destination="columns" after="main"/>
```

**Comment by vovayatsyuk** [comment]
I fixed this issue in this commit: https://github.com/breezefront/theme-frontend-breeze-blank/commit/21e966a3318dbd1ea65ae6b21381ff534d5767bf
